src/cmds/netspace.py

```diff
@@ -1,63 +1,27 @@
+import click
 import aiohttp
 import asyncio
-import time
-from time import struct_time, localtime
 from src.util.config import load_config
 from src.util.default_root import DEFAULT_ROOT_PATH
 from src.util.byte_types import hexstr_to_bytes
+from src.util.ints import uint16
 
 from src.rpc.full_node_rpc_client import FullNodeRpcClient
 
 
-def make_parser(parser):
-
-    parser.add_argument(
-        "-d",
-        "--delta-block-height",
-        help="Compare a block X blocks older."
-        + "Defaults to 192 blocks (~1 hour) and Peak block as the starting block."
-        + "Use --start BLOCK_HEIGHT to specify starting block",
-        type=str,
-        default="192",
-    )
-    parser.add_argument(
-        "-s",
-        "--start",
-        help="Newest block used to calculate estimated total network space. Defaults to Peak block.",
-        type=str,
-        default="",
-    )
-    parser.add_argument(
-        "-p",
-        "--rpc-port",
-        help="Set the port where the Full Node is hosting the RPC interface."
-        + "See the rpc_port under full_node in config.yaml. Defaults to 8555",
-        type=int,
-    )
-    parser.set_defaults(function=netspace)
-
-
-def human_local_time(timestamp):
```

Comment:
Removed this because wasn't used anywhere in the source code (I guess is not expected to be used by any external package).